index.html

```diff
@@ -10,38 +10,55 @@
 </head>
 
 <body>
-
+    <section class="main-container">
     <header class="main-header">
-        <nav class="nav-bar">
+        <nav class="nav-bar" >
             <div class="nav-img">
                 <img src="images/image_geometry_1.png" alt="geometry -img">
             </div>
             <div class="hamburger">
-                <img src="images/ic_menu.jpg" alt="hamburger">
+                <span class="line"></span>
+                <span class="line"></span>
+                <span class="line"></span>
+            </div>
+            <div class="hamburger-menu">
+                <ul>
+                    <li><a href="#">Hello</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
             </div>
 
         </nav>
     </header>
     <section class="homepage">
-        
+                 
         <div class="homepage-container">
             <h1 class="homepage-heading">Emmanuel <br>
                 Tofa</h1>
-            <p class="homepage-description1">Hello! I am a software developer! I can help you build a product, feature
+            <p class="homepage-description1">Hello! I am a Software Developer! I can help you build a product, feature
                 or website. Take a look at my work.</p>
             <p class="homepage-description2"> If you like what you see and have a project you need coded, don't hesitate
                 to contact me.</p>
                 <div class="btn-container">    
                     <button type="submit" class="homepage-start-collab">start collaboration</button>
                 </div>
                 <div class="homepage-container-img"></div>
-        .
+                <div class="dotted-square"></div>
+                
+                
     </section>
     <section class="projects">
```

Comment:
- [Optional] Could you make sure that your project card aligns perfectly as in the Figma template? You are expected to stick to design as much as possible.
- [Optional} Also kindly fix the font size and paddings of the button and list of technologies to match given template.
### Your design
![image](https://user-images.githubusercontent.com/94551484/222869373-1f490ceb-41d3-4118-85e2-904728e92327.png)

### FIgma template
![image](https://user-images.githubusercontent.com/94551484/222869217-c9d4be53-53ae-4059-aa1c-03f4bbb3f94f.png)